eclair-core/src/test/scala/fr/acinq/eclair/db/AuditDbSpec.scala

```diff
@@ -182,13 +183,20 @@ class AuditDbSpec extends AnyFunSuite {
     }
   }
 
-  test("handle migration version 1 -> 5") {
-    forAllDbs {
-      case _: TestPgDatabases => // no migration
-      case dbs: TestSqliteDatabases =>
-        import fr.acinq.eclair.db.sqlite.SqliteUtils.getVersion
-        val connection = dbs.connection
+  test("migrate sqlite audit database v1 -> v5/v6") {
```

Comment:
nit: why `v5/v6`, it's migrating to v5 since there's no v6?

Reply:
Done with aba59ab.